scripts/security/whitesource_container_scan.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+REPO_ROOT=$PWD/../..
+source $REPO_ROOT/solutions/kubernetes/functions.sh
+AGENT_TAR_LOCATION=/root/ws-k8s-agent/helm-chart/values.yaml
+#Download WhiteSource Plugin 
+wget http://cortx-storage.colo.seagate.com/releases/cortx/security/whitesource/ws-k8s-agent.tar && tar -xvf ws-k8s-agent.tar >/dev/null 2>&1
+
+function default_parameter() {
+    if [ -z "$WHITESOURCE_SERVER_URL" ]; then echo "WHITESOURCE_SERVER_URL not provided. Using default: https://saas.whitesourcesoftware.com ";WHITESOURCE_SERVER_URL=https://saas.whitesourcesoftware.com; fi
+    if [ -z "$USER_KEY" ]; then echo "USER_KEY not provided. Using default: 9b7231edebf249e6a0fee2c7d03a0e9fbead7e1ec81d463f9fa0fa988de6a07a";USER_KEY=9b7231edebf249e6a0fee2c7d03a0e9fbead7e1ec81d463f9fa0fa988de6a07a; fi
```

Comment:
Please remove the hardcoded key from the codebase. We will take this as user input in the Jenkins job. Exit out from script if `USER_KEY` is not provided

Reply:
function default_parameter() {
    if [ -z "$WHITESOURCE_SERVER_URL" ]; then echo "WHITESOURCE_SERVER_URL not provided. Using default: https://saas.whitesourcesoftware.com ";WHITESOURCE_SERVER_URL=https://saas.whitesourcesoftware.com; fi
    if [ -z "$USER_KEY" ]; then echo "USER_KEY not provided. Exiting script"; exit 1; fi
    if [ -z "$DOCKER_REGISTRY" ]; then echo "DOCKER_REGISTRY not provided. Using default: ghcr.io/seagate"; DOCKER_REGISTRY=ghcr.io/seagate; fi
echo -e "\n\n########################################################################"
   echo -e "# WHITESOURCE_SERVER_URL     : $WHITESOURCE_SERVER_URL              "
   echo -e "# DOCKER_REGISTRY            : $DOCKER_REGISTRY                     "
   echo -e "#########################################################################"
}
default_parameter